README.md

```diff
@@ -71,6 +71,13 @@ Hosting for Moonlight's Debian and L4T package repositories is graciously provid
 ### Steam Link Build Requirements
 * [Steam Link SDK](https://github.com/ValveSoftware/steamlink-sdk) cloned on your build system
 * STEAMLINK_SDK_PATH environment variable set to the Steam Link SDK path
+**Steam Link Hardware Limitations**  
+Moonlight builds for Steam Link are subject to hardware limitations of the Steam Link device. These limitations cannot be bypassed or changed by Moonlight:
+
```

Comment:
For GitHub to properly display this, I think you need to remove the empty space on line 76 and move it to line 74.